Documentation/Storage-Configuration/Monitoring/ceph-monitoring.md

```diff
@@ -194,9 +199,9 @@ The dashboards have been created by [@galexrt](https://github.com/galexrt). For
 
 The following Grafana dashboards are available:
 
-* [Ceph - Cluster](https://grafana.com/grafana/dashboards/2842)
-* [Ceph - OSD (Single)](https://grafana.com/grafana/dashboards/5336)
-* [Ceph - Pools](https://grafana.com/grafana/dashboards/5342)
+- [Ceph - Cluster](https://grafana.com/grafana/dashboards/2842)
```

Comment:
Did your editor change these? In Rook docs we are mostly using the `*` for bullet points for consistency

Reply:
Yes, Mrkdown lint extension changed it I think.